build-system/babel-config/pre-closure-config.js

```diff
@@ -39,6 +39,7 @@ function getPreClosureConfig() {
   ];
   const replacePlugin = getReplacePlugin();
   const preClosurePlugins = [
+    'optimize-objstr',
```

Comment:
According to https://www.npmjs.com/package/babel-plugin-optimize-objstr, should this be `optimize-obj-str`?

Reply:
Docs are wrong. (I'll update the NPM package as well).